packages/taco/src/conditions/base/json-api.ts

```diff
@@ -1,39 +1,21 @@
-import { JSONPath } from '@astronautlabs/jsonpath';
-import { z } from 'zod';
-
 import { Condition } from '../condition';
-import { OmitConditionType, returnValueTestSchema } from '../shared';
-
-export const JsonApiConditionType = 'json-api';
-
-const validateJSONPath = (jsonPath: string): boolean => {
-  try {
-    JSONPath.parse(jsonPath);
-    return true;
-  } catch (error) {
-    return false;
-  }
-};
-
-export const jsonPathSchema = z
-  .string()
-  .refine((val) => validateJSONPath(val), {
-    message: 'Invalid JSONPath expression',
-  });
-
-export const JsonApiConditionSchema = z.object({
-  conditionType: z.literal(JsonApiConditionType).default(JsonApiConditionType),
-  endpoint: z.string().url(),
-  parameters: z.record(z.string(), z.unknown()).optional(),
-  query: jsonPathSchema.optional(),
-  returnValueTest: returnValueTestSchema, // Update to allow multiple return values after expanding supported methods
-});
+import {
+  JsonApiConditionProps,
+  jsonApiConditionSchema,
+  JsonApiConditionType,
```

Comment:
What's the criteria for capitalizing some elements and others not?

Reply:
I'm unsure if there were criteria, but there was a pre-existing format for earlier conditions where the schema definition was *not* capitalized, but other types were.